### ISSUE TITLE
Bad u band coaddition on MACSJ2243 

### ISSUE THREAD

**Issue by boutigny** [LSSTDESC/ReprocessingTaskForce#3]
Image coaddition of u band on  MACSJ2243 is bad (see attached image). Probably due to poor astrometry. Need to check whether a simultaneous astrometric fit is improving the things.
![macsj2243_u_0_3 8](https://cloud.githubusercontent.com/assets/5713323/11722860/2e1ef7ce-9f20-11e5-9ef6-3bbde76994c2.png)

**Comment by PierreAstier** [comment]
Le 10/12/2015 18:27, Dominique Boutigny a écrit :

> Image coaddition of u band on MACSJ2243 is bad (see attached image). 
> Probably due to poor astrometry. Need to check whether a simultaneous 
> astrometric fit is improving the things.
> macsj2243_u_0_3 8 
> https://cloud.githubusercontent.com/assets/5713323/11722860/2e1ef7ce-9f20-11e5-9ef6-3bbde76994c2.png
> 
> —
> Reply to this email directly or view it on GitHub 
> https://github.com/DarkEnergyScienceCollaboration/ReprocessingTaskForce/issues/3.
> 
> Hello Dominique,

  I am seeing several instances of the same object on the image.
This may mean that the WCS's are so poor that the initial match
of the simultaneous astrometry (which assumes WCS's to ~ 2")
will not work. If I am right, the WCS's have to
be fixed beforehand.

Pierre.

## 

---

Pierre Astier , LPNHE, 12-22 1er étage.
4 place Jussieu, F 75252 Paris Cedex 05
tel (33) 1 44 27 76 47  ---- fax (33) 1 44 27 46 38

**Comment by idellant** [comment]
Hi Dominique,
sorry, I've been away for the last 10 days, but I'm back now.  I'm puzzled
by the image, though--because only part of the image shows multiple sources
(the galaxies at the top don't have the same level of error).  That
suggests that it's not an overall astrometric error--could it be that the
astrometric catalog is using an object with anomalous proper motion in the
region where the images are multiple?  That would cause swarp to build the
wrong projection just in that region and then the coadd would be o.k.
except that region...

Ian

**Comment by boutigny** [comment]
I found 2 issues:
- There is a very bright star close to the bad coadded region, this is screwing up the astrometry and the background estimation. But removing the ccd with the bright star, does not solve the multiple shiffted sources
- I tried to run the simultaneous astrometry fit on the u band before coadding. And it works (see attached image). There are no more multiple sources. 
  Well, this is still not perfect as the simultaneous fit is not able to remove all the outliers (factorization failed). I suppose that it may be due to a too small number of sources.
  ![macsj2243_u_0_3 8_simastrom](https://cloud.githubusercontent.com/assets/5713323/11731534/df7199c0-9f50-11e5-9a71-9e7535e1fed9.png)

**Comment by boutigny** [comment]
After thinking, Pierre and I realized that there is no way the simultaneous fit can re-associate sources that are so much separated. So here is the true story :
- The regular astrometry (ccd / ccd) did a poor job on these u band exposures, but the astromeric solution was good enough in the center of the exposure (it is completly off near the borders). 
- The simultaneous astrometry which is run after the initial astrometry probably associated the stars in the center and ignored the ones near the borders.
- These central stars were good enough to recover a satisfactory astrometric solution. But we were quite lucky on this one !

After some more debugging I finally found that the individual astrometry  was quietly failing in the SIP fitting when the SIP polynomial order was set to 4. By decreasing the polynomial order to 3, the fit is now ok. A DM JIRA Ticket ( DM-3549 / DM4640)  as been created.
